sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/CatalystConf.scala

```diff
@@ -22,6 +22,8 @@ import org.apache.spark.sql.catalyst.analysis._
 private[spark] trait CatalystConf {
   def caseSensitiveAnalysis: Boolean
 
+  def orderByOrdinal: Boolean
```

Comment:
Doesn't seem like a great name ... if we can come up with a good name I think having a single config makes sense. If not, doesn't matter that much to have two.